core/node/eth_watch/src/tests/mod.rs

```diff
@@ -228,18 +237,18 @@ async fn test_normal_operation_upgrade_timestamp() {
             (
                 ProtocolUpgrade {
                     version: ProtocolSemanticVersion {
-                        minor: ProtocolVersionId::next(),
+                        minor: ProtocolVersionId::Version28,
```

Comment:
Should be changed back to `::next()` in sync-layer-stable. For now, we need some version which is > `gateway_upgrade`